fuzzers/machxo2/021-glb-entry/fuzzer.py

```diff
@@ -0,0 +1,40 @@
+from fuzzconfig import FuzzConfig
+import interconnect
+import pytrellis
+
+jobs = [
+    {
+        "cfg": FuzzConfig(job="GLB_ENTRY", family="MachXO2", device="LCMXO2-1200HC", ncl="tap.ncl",
+                          tiles=["CENTER6:CENTER_EBR_CIB"]),
+        "left_net": "R6C{}_HPSX{:02d}00",
+        "right_net": "R6C{}_HPSX{:02d}00"
+    },
+]
+
+
+def main():
+    def left_end(x):
```

Comment:
I presume these functions are 1200-specifc? Would be a good idea to leave a note about that here.

Reply:
This should be resolved by 8b7845f.